Index: FinalProject/edu/ucalgary/oop/CleaningTask.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class CleaningTask {\r\n    private int cleanTime;\r\n\r\n    public CleaningTask(String name) {\r\n        if (name.equalsIgnoreCase(\"c\") || name.equalsIgnoreCase(\"f\") || name.equalsIgnoreCase(\"r\")) {\r\n            cleanTime = 5;\r\n        } else if (name.equalsIgnoreCase(\"p\")) {\r\n            cleanTime = 10;\r\n        } else {\r\n            System.out.println(\"Invalid animal name for cleaning task.\");\r\n        }\r\n    }\r\n\r\n    public int getCleanTime() {\r\n        return cleanTime;\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/FinalProject/edu/ucalgary/oop/CleaningTask.java b/FinalProject/edu/ucalgary/oop/CleaningTask.java
--- a/FinalProject/edu/ucalgary/oop/CleaningTask.java	(revision 9179f8629d4958c144703de57415ff306356e725)
+++ b/FinalProject/edu/ucalgary/oop/CleaningTask.java	(date 1680563114576)
@@ -1,3 +1,5 @@
+package FinalProject.edu.ucalgary.oop;
+
 public class CleaningTask {
     private int cleanTime;
 
Index: FinalProject/edu/ucalgary/oop/FeedingTask.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class FeedingTask {\r\n    private int numberAnimal;\r\n    private int feedTime;\r\n\r\n    public void FeedingTask(String name){\r\n\r\n    }\r\n\r\n    public int getFeedTime() {\r\n        return feedTime;\r\n    }\r\n\r\n    public int getNumberAnimal() {\r\n        return numberAnimal;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/FinalProject/edu/ucalgary/oop/FeedingTask.java b/FinalProject/edu/ucalgary/oop/FeedingTask.java
--- a/FinalProject/edu/ucalgary/oop/FeedingTask.java	(revision 9179f8629d4958c144703de57415ff306356e725)
+++ b/FinalProject/edu/ucalgary/oop/FeedingTask.java	(date 1680564796486)
@@ -1,3 +1,5 @@
+package FinalProject.edu.ucalgary.oop;
+
 public class FeedingTask {
     private int numberAnimal;
     private int feedTime;
Index: FinalProject/edu/ucalgary/oop/Animal.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package FinalProject.edu.ucalgary.oop;\r\npublic class Animal{\r\n    public String name;\r\n    public int numanimal;\r\n    public int tobefed;\r\n    public int feedtime;\r\n    public int cleantime;\r\n    private FeedingTask feed;\r\n    private CleaningTask clean;\r\n    public animal(String name){\r\n        this.name=name;\r\n        this.feed = new FeedingTask(name);\r\n        this.clean = new CleaningTask(name);\r\n        this.numanimal =feed.getnumberanimals();\r\n        this.tobefed = feed.getnumberanimals();\r\n        this.feedtime = feed.getfeedtime();\r\n        this.cleantime = feed.getcleantime();\r\n\r\n    }\r\n    public void decTobefed() {\r\n        this.tobefed -= this.tobefed;\r\n    }\r\n\r\n    public int getTobefed() {\r\n        return tobefed;\r\n    }\r\n\r\n    public int getCleantime() {\r\n        return cleantime;\r\n    }\r\n\r\n    public int getFeedtime() {\r\n        return feedtime*tobefed;\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/FinalProject/edu/ucalgary/oop/Animal.java b/FinalProject/edu/ucalgary/oop/Animal.java
--- a/FinalProject/edu/ucalgary/oop/Animal.java	(revision 9179f8629d4958c144703de57415ff306356e725)
+++ b/FinalProject/edu/ucalgary/oop/Animal.java	(date 1680564796472)
@@ -7,14 +7,14 @@
     public int cleantime;
     private FeedingTask feed;
     private CleaningTask clean;
-    public animal(String name){
+    public void Animal(String name){
         this.name=name;
         this.feed = new FeedingTask(name);
         this.clean = new CleaningTask(name);
-        this.numanimal =feed.getnumberanimals();
-        this.tobefed = feed.getnumberanimals();
-        this.feedtime = feed.getfeedtime();
-        this.cleantime = feed.getcleantime();
+        this.numanimal =feed.getNumberAnimal();
+        this.tobefed = feed.getNumberAnimal();
+        this.feedtime = feed.getFeedTime();
+        this.cleantime = clean.getCleanTime();
 
     }
     public void decTobefed() {
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 9179f8629d4958c144703de57415ff306356e725)
+++ b/.idea/misc.xml	(date 1680563114773)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager">
+  <component name="ProjectRootManager" version="2" project-jdk-name="11" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
